### PR TITLE
Long sleep xsoar fix

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — COMMENTED** [review]
the config.yml changes are temp I guess.
other than that looks great.

**Comment by orhovy** [comment]
> the config.yml changes are temp I guess.
> other than that looks great.

Yes, temp , will revert them .
This branch will be used for the sdk release I guess.

File: Packs/McAfee_ESM/ReleaseNotes/1_1_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### McAfee ESM v2
+- Maintenance and stability enhancements.
```
Comment (bakatzir) [inline Packs/McAfee_ESM/ReleaseNotes/1_1_1.md:4]:
```suggestion
- Maintenance and stability enhancements.
- Upgraded the Docker image to demisto/python3:3.8.5.11789.
```

**Review by bakatzir — APPROVED** [review]
just add the rn regarding update of docker image

File: Packs/AutoFocus/ReleaseNotes/1_1_4.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### AutoFocus Feed
+- Maintenance and stability enhancements.
```
Comment (amshamah419) [inline Packs/AutoFocus/ReleaseNotes/1_1_4.md:4]:
```suggestion
- Maintenance and stability enhancements.
- Updated docker image from `3.8.5.10845` to `3.8.5.11789`.
```

**Pull request merged**